repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/bulletin-main.ftl

```diff
@@ -0,0 +1,56 @@
+<#include "../../include/imports.ftl">
+<#include "../macros/last-next-reviewed-date.ftl">
+<#-- @ftlvariable name="categoriesMap" type="java.util.Map" -->
+<#-- @ftlvariable name="item" type="uk.nhs.hee.web.beans.Bulletin" -->
+<#-- @ftlvariable name="pageable" type="org.onehippo.cms7.essentials.components.paging.Pageable" -->
+
+<#if pageable??>
+    <#list pageable.items as item>
+        <!-- TODO check if we can't put a getter inside the uk.nhs.hee.web.beans.Link to get the linkUrl,
+         so that we don't have to check if the link is internal or external -->
+        <#if item.link??>
+            <#if item.link.document??>
+                <@hst.link hippobean=item.link.document var="linkURL"/>
+                <#assign openInANewWindow=false />
+            <#else>
+                <#assign openInANewWindow=true />
+                <#assign linkURL="${item.link.url}" />
+            </#if>
+        </#if>
```

Comment:
@adaMiclos Do you think we can use the `<@link` macro here ?

Reply:
@thojava I have removed the Link compound type from the Bulletin document type because it will contain only external links.